hack/local-up-cluster.sh

```diff
@@ -326,11 +332,11 @@ function start_apiserver {
     # which should be able to be used as the CA to verify itself
 
     anytoken_arg=""
-    if [[ -n "${ALLOW_ANY_TOKEN:-}" ]]; then
+    if [[ "${ALLOW_ANY_TOKEN}" = true ]]; then
       anytoken_arg="--insecure-allow-any-token "
     fi
     authorizer_arg=""
-    if [[ -n "${ENABLE_RBAC:-}" ]]; then
+    if [[ "${ENABLE_RBAC}" = true ]]; then
```

Comment:
We don't require exact match to `true` for other `ALLOW_*/ENABLE_*` options, why these?